README.md

```diff
@@ -33,25 +86,169 @@ VS Code will generate a `launch.json` in your project with some default configur
 
 ![React Native launch targets](images/debug-targets.png)
 
-Once app is loaded and ran, [open Developer Menu](https://facebook.github.io/react-native/docs/debugging#accessing-the-in-app-developer-menu) inside your application and enable remote debugging by clicking on `Debug JS Remotely` button.
+Once app is loaded and running, [open the developer menu](https://reactnative.dev/docs/debugging#accessing-the-in-app-developer-menu) inside your application and enable remote debugging by clicking on `Debug JS Remotely` button.
 
 ![React Native enable remote debug](images/enable-remote-debug.png)
 
-You can debug your app on an Android emulator, Android device or iOS simulator, iOS device. Please see the documentation for [debugging on iOS devices](doc/debugging.md#debugging-on-ios-device).
+The extension allows you to debug multiple devices and configurations, please read the following sections for more information for your particular use case.
+
+## Hermes
+
+Hermes is the new JavaScript engine optimized for running React Native apps on Android. It improves app performance and decreases app size.
+
+Click [here](https://reactnative.dev/docs/hermes) to learn more about Hermes.
+
+Debugging apps with Hermes enabled is currently experimental. Please, see [this issue](https://github.com/microsoft/vscode-react-native/issues/1073) for current known issues on Hermes support.
+
+To debug while using Hermes engine, please choose one of the following debug configurations:
+ - React Native (Hermes): Debug Android - Experimental
+
+```json
+{
+    "name": "Debug Android (Hermes) - Experimental",
+    "cwd": "${workspaceFolder}",
+    "type": "reactnativedirect",
+    "request": "launch",
+    "platform": "android"
+}
+```
+
+ - React Native (Hermes): Attach to Hermes application - Experimental
+
+```json
+{
+    "name": "Attach to Hermes application - Experimental",
+    "cwd": "${workspaceFolder}",
+    "type": "reactnativedirect",
+    "request": "attach"
+}
+```
+
+## iOS devices
+
+Debugging on an iOS device require following manual steps:
+* Install [ios-deploy](https://www.npmjs.com/package/ios-deploy) `npm install -g ios-deploy`.
+* Have a valid iOS Development certificate installed.
```

Comment:
Should Development be capitalized here?

Reply:
```suggestion
* Have a valid iOS development certificate installed.
```

This step was before in another example. We should be consistent with the wording. I believe it was:

> Install a valid iOS development certificate.

I'm OK either way. What do people prefer?

Reply:
On the [official Apple page](https://developer.apple.com/support/certificates/), they often use the "Developer" with a capital letter, e.g. "Developer ID". So I suppose that the word "Development" could be capitalized.